src/backend/user/user.service.ts

```diff
@@ -102,6 +103,9 @@ export async function createUser(userDataFromGh: CreateUserPayload) {
       githubDetails: true,
     },
   })
+  await httpClient.post('http://localhost:3000/api/discord', {
```

Comment:
**issue:** localhost

Reply:
kind off not resolved! (now on profile.service)